packages/core/src/customizations/components/fragments/plp.ts

```diff
@@ -0,0 +1,9 @@
+import { gql } from '@faststore/graphql-utils'
+
+export const fragment = gql`
+  fragment plp on Query {
+    collection(slug: $slug) {
```

Comment:
nice! we need to think on how to resolve the page level queries for the other queries as well afterward. Like the `search` query from the `useGalleryQuery` hook as we talked.

Reply:
Isn't search a PLP?